src/main/kotlin/org/jetbrains/research/testspark/collectors/IntegratedTestsCollector.kt

```diff
@@ -0,0 +1,41 @@
+package org.jetbrains.research.testspark.collectors
+
+import com.intellij.internal.statistic.eventLog.events.EnumEventField
+import com.intellij.internal.statistic.eventLog.events.EventPair
+import com.intellij.internal.statistic.eventLog.events.IntEventField
+import com.intellij.internal.statistic.service.fus.collectors.CounterUsagesCollector
+import org.jetbrains.research.testspark.collectors.data.Technique
+import org.jetbrains.research.testspark.data.CodeType
+
+/**
+ * This class represents a collector for tracking integrated tests to the project events.
+ *
+ * @property count the IntEventField instance representing the count for the event
+ * @property technique the EnumEventField<Technique> instance representing the technique for the event
+ * @property level the EnumEventField<CodeType> instance representing the code type for the event
+ * @property modifiedTestsCount the IntEventField instance representing the modified tests count for the event
+ * @*/
```

Comment:
The 17th line still contains a redundant `@` symbol at the end of the Javadoc comment.